### PR TITLE
[#1501] Testing getMinMaxPointErrors in FeedbackConstantSumQuestionDetails

### PR DESCRIPTION
**Pull request by AlefMemTav**
Part of #1501

Test of multiple conditions.
<html>
<body>
<b style="font-weight:normal;" id="docs-internal-guid-cba7efef-7fff-4ddb-d9c0-3cde48a6d7e9"><div dir="ltr" style="margin-left:19.25pt;" align="left">

ID | Conditions | Input | Output
-- | -- | -- | --
CT1 | minPoint != null && answer < minPoint | minPoint = 5, answer = 4 | "An answer cannot be smaller than the minimum number of points: 5"
CT2 | minPoint != null && answer < minPoint | minPoint = 10, answer = 10 | False
CT3 | minPoint != null && answer < minPoint | minPoint = null, answer = 9 | False
CT4 | maxPoint != null && answer > maxPoint | maxPoint = 10, answer = 11 | "An answer cannot be greater than the maximum number of points: 10"
CT5 | maxPoint != null && answer > maxPoint | maxPoint = 10, answer = 10 | False
CT6 | maxPoint != null && answer > maxPoint | maxPoint = null, answer = 11 | False
CT7 | minPoint, maxPoint = null | answer = 8 | [] 

</div></b>
</body>
</html>

### Jacoco

jacoco

### CODE REVIEW

**Review by zhaojj2209 — CHANGES_REQUESTED** [review]
Thank you for your PR, however it does not follow our [contributing guidelines](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr). Your PR:

- Does not address any specific issue
- Does not follow our PR naming conventions (PR title should contain the issue number for the issue to be addressed, not the PR number itself)
- Fails linting

Additionally, please remove all changes that are not related to the PR (e.g. `build.gradle`, `MANIFEST.MF`, `EmailGeneratorTest`).

**Comment by AlefMemTav** [comment]
I don't know how to make these little corrections, so I'll leave it at that for now.

**Comment by AlefMemTav** [comment]
It's OK now?

**Comment by wkurniawan07** [comment]
@AlefMemTav as far as private methods are concerned, they are usually tested with the public/protected method calling them, unless they are complicated enough to warrant their own test cases.
In this case, you may first want to check, maybe the tests you have written are already part of test cases for `validateResponsesDetails`?

**Comment by AlefMemTav** [comment]
I checked and they are not part. See the Jacoco. I increased the coverage of the method.

File: src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetails.java [hunk 1]
```diff
@@ -242,7 +242,12 @@ private List<String> getErrorsForConstSumRecipients(List<FeedbackResponseDetails
         return getErrors(givenPoints, totalPoints);
     }
 
-    private List<String> getMinMaxPointErrors(List<Integer> answers) {
+    /**
+    * Takes a list of integers and returns a list of errors for integers above or below the value.
+    * @param answers List of integers
+    * @return errors List of integers
+    */
```
Comment (zhaojj2209) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetails.java:249]: This isn't really needed as the method itself is self-explanatory enough.

**Review by zhaojj2209 — COMMENTED** [review]
As has been mentioned, private methods are usually tested via the public/protected methods calling them. You may want to consider modifying your testcases to be part of the two `testValidateResponseDetails_amongOptions` testcases instead, that way `getMinMaxPointErrors` can be tested without having to change the method from `private` to `protected`.

**Comment by zhaojj2209** [comment]
@AlefMemTav Any updates on this PR?

**Comment by AlefMemTav** [comment]
Next week I finish! I'm out of time because of university.

**Pull request closed without merging**